lib/DojoAMDMainTemplatePlugin.js

```diff
@@ -107,9 +107,9 @@ module.exports = class DojoAMDMainTemplatePlugin {
 				buf.push("\tc: createContextRequire,");
 				buf.push("\tm: dojoModuleFromWebpackModule,");
 				buf.push("\th: resolveTernaryHasExpression,");
-				buf.push("\tg: (function(){return this;})()   // Easy access to global scope");
+				buf.push("\tg: (function(){return this || window;})()   // Easy access to global scope");

```

Comment:
What value does `this` have when it is falsy?  What environment are you running in?  I have a hard time understanding how `this` would not have a value.

Reply:
Its undefined.  Which is because its ultimately inside a "use strict" section (I think)?

Reply:
What environment?

Reply:
Also, none of the bootstrap code (including the code you modified above) is placed inside a "use strict" section by webpack.  Is something else you're using wrapping the entire app within a "use strict"?

Reply:
Like I mentioned in the issue, the problem happens when this bundle is in turn bundled a second time (in rollupjs for example).  

I will try and get a sample project uploaded to GitHub by end of week.